### PR TITLE
Updated sketchUp

### LINKED ISSUE #6570: Update SketchUp
I'm submitting an ISSUE: please check one with "x"

- [x] New question{'s} 
Screenshot 2023-10-19 at 4 17 20 PM
Screenshot 2023-10-19 at 4 17 27 PM

### PR DESCRIPTION
**Pull request by tuhinaww**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [x] I have added new quiz{'s}
- [x] I have made small correction/improvements

### Changes made
Screenshot 2023-10-19 at 4 17 20 PM
Screenshot 2023-10-19 at 4 17 27 PM

Closes #6570

### CODE REVIEW

File: sketchup/sketchup-quiz.md [hunk 1]
```diff
@@ -188,3 +188,59 @@
...
+
+- [ ] a) It's built-in and doesn't require an extension
+- [ ] b) SketchUp OBJ
+- [ ] c) SketchUp 3DS
+- [x] d) SketchUp STL
+
+#### Q35. How do you save your work in SketchUp?
+
+- [ ] a) Click File > Save
+- [ ] b) Use the Export tool
```
Comment (haxkd) [inline sketchup/sketchup-quiz.md:244]: why not option (a)

**Review by haxkd — CHANGES_REQUESTED** [review]

File: sketchup/sketchup-quiz.md (hunk 1 above)
Comment (haxkd) [reply sketchup/sketchup-quiz.md:244]: Im saying that the correct answer can be [a] and [c] too.

Comment (Ebazhanov) [reply sketchup/sketchup-quiz.md:244]: guys, from chat gpt 
image

Comment (Ebazhanov) [reply sketchup/sketchup-quiz.md:244]: could be two option, but if linkedin quiz accept only one, then the answer is only [a]

**Review by Ebazhanov — APPROVED** [review]
Thank you! 👍

**Pull request merged**